<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Top10Menu</class>
 <widget class="QMainWindow" name="Top10Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#menuButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#menuButton{background-color:rgba(0,0,0,0); border-radius:15px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listTop10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTuto">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{ color: white;}</string>
    </property>
    <property name="text">
     <string>Niveau Tuto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLevel1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{ color: white;}</string>
    </property>
    <property name="text">
     <string>Level 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLevel2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{ color: white;}</string>
    </property>
    <property name="text">
     <string>Level 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
